docs/en/pid-eaa-issuance.rst

```diff
@@ -726,43 +730,44 @@ Token endpoint response MUST contain the following mandatory claims.
       - Expiry time of the *Access Token* in seconds.
       - :rfc:`6749`.
     * - **c_nonce**
-      - JSON string containing a ``nonce`` value to be used to create a *proof of possession* of key material when requesting a credential. 
+      - JSON string containing a ``nonce`` value to be used to create a *proof of possession* of key material when requesting a Credential.
```

Comment:
If I'm not mistaken, Credential is not a defined term. I see only Digital Credential. Probably we should say in the defined term Section that Credential refers to Digital Credential. 
@peppelinux WDYT?

Reply:
The text that defines the term DigitaleCredential also says 

> It is also known as Credential.